### DISCUSSION TITLE
Send Automatic Recurring Payments on Venmo using Python + GitHub Actions

### DISCUSSION THREAD

**Discussion by jsjoeio** [mmohades/Venmo#39] [Show and tell]
I used this library + [notifiers](https://github.com/liiight/notifiers) + GitHub Actions to send automatic recurring payments on Venmo! How it works:
- the users id's of 3 friends are stored in environment variables
- once a month, the script runs on a schedule using GitHub Actions
- it sends 3 requests for money and I have a callback which sends a message on Telegram letting me know it worked
- I have a second script which runs once a week to make sure everything is still working

### Learn more
- [Repo](https://github.com/jsjoeio/monthly-venmo) 
- [Video](https://www.youtube.com/watch?v=cMHORRmHDJs)
- [Article](https://joeprevite.com/send-automatic-recurring-payments-on-venmo)

Thank you @mmohades for this awesome library! 🙌

**Comment by mmohades** [comment]
This is cool, I liked the Telegram notifications part by Efron lol. Thanks for sharing!
